plotsearch/models.py

```diff
@@ -118,4 +119,24 @@ class PlotSearchTarget(models.Model):
     )
 
 
+class TargetInfoLink(models.Model):
+    """
+    In Finnish: Lisätietolinkki
+    """
+
+    # In Finnish: Tonttihaun kohde
+    plot_search_target = models.ForeignKey(
+        PlotSearchTarget, on_delete=models.CASCADE, related_name="info_links", null=True
```

Comment:
What is point if plot_search_target = None?

Reply:
Now that you said it, there's actually none.
Original point was that you are able to add links without binding them to plot search targets and bind them afterwards. But now when I'm thinking, there should not be that kind of cases ever.